Rovers/Rovers.tex

```diff
@@ -0,0 +1,388 @@
+This is also an area that RIT
+Space Exploration has very little experience with project-wise.
+The purpose of this project is to assess and assert the capability
+of RIT SPEX regarding the construction and fabrication of a mock-rover and then begin construction. 
+As this is a new area to RIT SPEX, prototyping and experimenting is necessary to most effectively accomplish this goal. 
+This project will look at the unknowns, technical challenges, project management, and member skills of RIT Space Exploration in regard to a University Rover Competition (URC) capable rover. 
+The URC is hosted by The Mars Society annually. 
+It is very competitive and requires a very complex and expensive rover that is currently out of scope for SPEX. 
+This rover would need to be capable of full autonomous navigation, soil collection and analysis, and precise robotics. 
+This project is the first step in building the skills for such a team.
+
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+
+The goal of this project is delivering a functioning mock rover.
+We define mock rover as a functioning rover built to specifications
+for a mock mission as it will not be launched. The rover
+will allow the team to develop skills and knowledge to build a
+more feature completer rover in the future. These skills include
+rover fabrication and design, autonomous navigation, electrical
+system design, rover testing, computer vision, and robotics.
+The rover will be testing using events designed like
+the URC but scaled down to fit the scope of this rover. We
+will be using time as a metric as well as binary metrics for
+completion of the parts of the competition. This includes ’does
+the rocker-bogie system works as intended’, ’will the rover stop
+when directed at an obstacle’, etc. The details are defined in
+the minimum viable product section.
+
+\begin{table}[hb!]
+    % the "h" in these brackets tells LaTeX to put the table Here. Try [t] for top and [b] for bottom,
+    % or [hbp] for "here, or if you can't do that put it at the bottom of the page, or if you can't do that put it on its own page.
+    % Here we've also used an "!" to yell at LaTeX to DO THIS OR ELSE!
+    \caption{Minimum Viable Product}
+    \centering
+    {\renewcommand{\arraystretch}{1.2}
+    \begin{tabularx}{\linewidth}{LL} 
+    \hline
+    \textbf{Feature} & \textbf{Reason} \\
+    \hline
+    Rocker-bogie-like suspension & This is the NASA perfered rover suspension style. \\
```

Comment:
We make a lot of concessions with mock ups and demos for NASA-like projects that diverge from NASA's preferences. Why choose this suspension style? Are you aiming for high performance or Mars like terrain? Are you aiming to mimic Curiosity? (Both are valid arguments in my opinion) or is there a halfway point that you can go for?